backend/app/schemas/idioms.py

```diff
@@ -0,0 +1,9 @@
+import uuid
+import sqlmodel
+
+
+class Idiom(sqlmodel.SQLModel, table=True):
+    id: uuid.UUID | None = sqlmodel.Field(default=None, primary_key=True)
+    text: str = sqlmodel.Field(index=True)
+    meaning: int | None = sqlmodel.Field(default=None, index=True)
+    secret_name: str
```

Comment:
This is not the final structure, I will adapt this to our current data structures in `languages/english/root.json`